examples/spaceships/assets/fonts/quicksand-light.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2011 The Quicksand Project Authors (https://github.com/andrew-paglinawan/QuicksandFamily), with Reserved Font Name “Quicksand”.
```

Comment:
Is this needed? we can probably do without this in the example

Reply:
let me figure out if i can use a bevy built-in font to avoid bundling this.. i found the player labels to be really useful when debugging so don't want to forgo labels entirely.

Reply:
changed to using the built-in font and removed this